tests/framework/installer/ceph_manifests.go

```diff
@@ -82,7 +83,11 @@ func (m *CephManifestsMaster) GetOperator() string {
 	} else {
 		manifest = m.settings.readManifest("operator.yaml")
 	}
-	return m.settings.replaceOperatorSettings(manifest)
+	manifest = m.settings.replaceOperatorSettings(manifest)
+
+	// In release branches replace the tag with a master build since the local build has the master tag
+	r, _ := regexp.Compile(`image: rook/ceph:v[a-z0-9.-]+`)
```

Comment:
```suggestion
	r, _ := regexp.Compile(`image: rook/ceph:v[0-9.-]+`)
```

Reply:
The tag can have `beta` or similar, so this was needed.